images/base/entrypoint

```diff
@@ -18,6 +18,19 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+configure_proxy() {
+  # Don't use proxy for the local network
+  LOCAL_NETWORK=$(ip route list dev eth0 proto kernel | awk '{ print $1 }')
+
+  mkdir -p /etc/systemd/system/containerd.service.d/
+  cat <<EOF >/etc/systemd/system/containerd.service.d/http-proxy.conf
```

Comment:
we should really be configuring more than just containerd, working on an alternate fix

Reply:
agree